### DISCUSSION TITLE
flask debug (auto reload) feature not working

### DISCUSSION THREAD

**Discussion by alshukri9** [pallets/flask#5220] [Q&A]
the issue when i create docker container for the app with dockerfile
ENTRYPOINT [ "python3", "-m", "flask", "run", "--debug", "--host=0.0.0.0"]
everything works fine but when i change anything in the code i don't see anything change if i wanna see the changes
i have to rebuild the container

note: i am using correct volume
and when i exec the container i can see the changes in the file

![hsdf](https://github.com/pallets/flask/assets/129284063/962ffd93-f077-45ce-837b-ef05d48f6ced)

Environment:

- Python version: 3.10
- Flask version: 2.2.3

**Comment by ThiefMaster** [comment]
Not sure how you expect that to work. You're copying the code into the container. This happens only at build time. Using the reloader there is pointless.

What you need to do is mounting your code as a volume into the container (and get rid of the `COPY`). Then the reloader will work.

**Reply by alshukri9** [reply]
if u notice i have said i am using correct volume (yes i am using copy in dockerfile and volume in docker-compose)
the code is changing in the docker container
